react/components/EXPERIMENTAL_Modal/__snapshots__/index.test.tsx.snap

```diff
@@ -492,7 +492,7 @@ exports[`Modal isOpen Should show modal when isOpen is true and hide when is fal
     <div
       aria-labelledby="vtex-modal__title"
       aria-modal="true"
-      class="vtex-modal__modal flex flex-column relative bg-white shadow-5 center mv9 br2 undefined vw-90"
+      class="vtex-modal__modal flex flex-column relative bg-white shadow-5 center mv9 br2 undefined undefined vw-90"
```

Comment:
Do you know what are those `undefined`?

Reply:
In unit tests the style imported into the component is undefined for some reason :thinking:

Reply:
maybe the jest configuration isn't properly setup to handle css modules?